GNUmakefile

```diff
@@ -61,17 +68,17 @@ testacc: fmtcheck
 debugacc: fmtcheck
 	TF_ACC=1 dlv test $(TEST) --headless --listen=:2345 --api-version=2 -- -test.v $(TESTARGS)
 
+website-lint:
+	@echo "==> Checking website against linters..."
+	@misspell -error -source=text -i hdinsight website/
+
```

Comment:
alphabetically this one wants to be below the other one?

Reply:
yestrue but aesthetically and functionally I think it's better above given the last two massive blocks run together and i spent a while skimming the file looking for it